server/test/mockData/attainable.ts

```diff
@@ -0,0 +1,78 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { AttainableData } from '../../src/types/course';
+
+export const mockAttainables: AttainableData = {
+  name: '2 optional exercises',
+  executionDate: new Date(2023, 9, 1),
+  expiryDate: new Date(2024, 8, 14),
+  subAssignments: [
+    { name: 'Exercise 1',
```

Comment:
The client code does use this style, but it would be more consistent with our server code to put a newline after the opening bracket here:
```ts
subAssignments: [
  {
    name: 'Exercise 1',
    ...
  },
]
```